src/py42/clients/securitydata.py

```diff
@@ -102,10 +102,10 @@ def get_all_plan_security_events(
                         - ``PERSONAL_CLOUD_SCAN_RESULT``
 
                     Defaults to None.
-            min_timestamp (float, optional): A POSIX timestamp representing the beginning of the
-                date range of events to get. Defaults to None.
-            max_timestamp (float, optional): A POSIX timestamp representing the end of the date
-                range of events to get. Defaults to None.
+            min_timestamp (int or str or datetime, optional): Timestamp in milliseconds or
```

Comment:
Looks like you removed `float` here, we should keep float since we supported it before, and I think one of our util methods returns a float.. You can keep `int` as well, .... so it should be `float or int or str or datetime`

Reply:
Added.